pkg/monitoring/watcher_windows.go

```diff
@@ -0,0 +1,17 @@
+// +build windows
+
+package monitoring
+
+import (
+	"github.com/cloudradar-monitoring/cagent/perfcounters"
+)
+
+var watcher *perfcounters.WinPerfCountersWatcher
```

Comment:
Again, can we avoid globals?

Reply:
It can be done but probably not within given PR. Some few other objects need this and to do it properly `Cagent` must allocate it and pass during `New` or `Configure` for `Top` `VM` etc.
But that will need to hide context of watcher type to `interface` as only windows stuff uses windows perfcounters